test-dataset-plan.md

```diff
@@ -0,0 +1,39 @@
+# Plan for the test dataset
+
+We want the test dataset to contain an equal amount of data split into various
+tasks, described below.
+
+Aside from ensuring that the resulting models perform well across multiple
+tasks and do not overfit to any single one such, this would also provide
+transparent performance descriptions of the trained models. Such transparency
+would mean that anyone who is looking for a language model to finetune on their
+dataset can immediately see whether the model works well on their domain/task.
+
+## Tasks
+
+As mentioned above, we want the test dataset to contain an equal amount of
+data from each of the following tasks:
+
+- Danish web data (Netarkivet)
+- Danish social media data (HOPE Twitter, Reddit-da and Hestenettet from Danish
```

Comment:
I imagine Twitter data is quite different from Hestenettet, and that some applications would look into Twitter without ever touching something like Hestenettet. 

Perhaps it'd be helpful to report results for Twitter/Hestenettet (etc., whatever our lowest level of granularity is) separately?

Reply:
In that case I'd vote for just removing Hestenettet, as I don't see the merit of it on its own.

Reply:
+1 for separating Twitter. Fine with removing Hestenettet

Reply:
I agree as well. @saattrupdan it should be possible to create the dataset downloader such that it rehydrates the tweets when downloading it (at least from what I can see in the documentation you can use the `use_auth_token` (or potentially even the **kwargs) to supply an optional Twitter token.